.gitignore

```diff
@@ -1,12 +1,13 @@
 _site/
 .asset-cache
-.sass-cache
-.jekyll-cache
-# From debugging
-.bash_history
-node_modules
-.gemfiles/
 .bundle/
+.gemfiles/
+.gems
+.jekyll-cache
 .jekyll-metadata
+.sass-cache
+assets
```

Comment:
Do we need to exclude the entire assets folder ? This contains all the css/js/images

Reply:
Good catch. I had missed that.

The reason I added it was because running the build script caused a file to be added. I'll see if I can trim the path down to be more specific.

Reply:
It looks like the build process is adding files to assets/resized.

Reply:
They then get copied to <built site>/assets/resized.

Reply:
This is due to the current Jekyll image plugin that we're using (jekyll-responsive-image) requiring an output path for resized images. For now just add assets/resized to the .gitignore. This will change once I've migrated sites over to using the jekyll-picture-tag plugin for webp image support.